### DISCUSSION TITLE
Input training shape for YOLOv8(IMGSZ)

### DISCUSSION THREAD

**Discussion by ismailkattar** [ultralytics/ultralytics#16155] [Q&A] [answered]
Hello folks,

Is there an optimal image shape for training YOLOv8, or does it depend on the overall distribution of my image shapes? Most instance segmentation and object detection projects I’ve reviewed use an image size of 640. Is 640 a standard setting? (Note: I read that the image size should be a multiple of 32 is it right.)

**Comment by glenn-jocher** [comment]
The image size of 640 is commonly used for YOLOv8, and yes, it should be a multiple of 32 to ensure compatibility with the model's architecture. Adjusting the size can depend on your specific dataset and computational resources.

**Reply by ismailkattar** [reply]
so using 1024 image size is applicable but it will cost more resources right?

**Reply by glenn-jocher** [reply, marked as answer]
@ismailkattar yes, using a 1024 image size is possible, but it will require more computational resources and memory.
